src/charm.py

```diff
@@ -97,6 +98,13 @@ def __init__(self, *args: Any):
             extra_user_roles="SUPERUSER",
         )
 
+        self.ingress_per_unit = IngressPerUnitRequirer(
```

Comment:
Just have a question:

Do we need to subscribe the ready and revoked events to update the `ldap` integration databag if there are any existing `ldap` integrations?

Reply:
Hmm... we likely should handle those events as this will change the address requirers utilize to contact the Glauth server. Simple `netcat` scan indicates that other pods can communicate with the open port on the on the load balancer:

```shell
# 10.0.0.135 is the address that traefik-k8s grabbed from MetalLB.
root@loki-0:/# nc -zv 10.0.0.135 3893
Connection to 10.0.0.135 3893 port [tcp/*] succeeded!
```

We'll need to check if any requirers have joined the `ldap` relation and update the databag.